indra/newview/llviewermenu.cpp

```diff
@@ -9617,7 +9680,10 @@ void initialize_menus()
     view_listener_t::addMenu(new LLAdvancedClickRenderShadowOption(), "Advanced.ClickRenderShadowOption");
     view_listener_t::addMenu(new LLAdvancedClickRenderProfile(), "Advanced.ClickRenderProfile");
     view_listener_t::addMenu(new LLAdvancedClickRenderBenchmark(), "Advanced.ClickRenderBenchmark");
+    view_listener_t::addMenu(new LLAdvancedClickHDRIPreview(), "Advanced.ClickHDRIPreview", cb_info::UNTRUSTED_BLOCK);
+    view_listener_t::addMenu(new LLAdvancedClickGLTFScenePreview(), "Advanced.ClickGLTFScenePreview", cb_info::UNTRUSTED_BLOCK);
```

Comment:
These new items `Advanced.ClickHDRIPreview`, `Advanced.ClickGLTFScenePreview` and `Advanced.RebuildTerrain` didn't have `cb_info` enum values. I guessed `UNTRUSTED_BLOCK`, but please suggest what we should use instead.